examples/tutorials/flare_api.ipynb

```diff
@@ -160,8 +159,31 @@
    },
    "outputs": [],
    "source": [
-    "! pip install -r ./requirements.txt\n",
-    "! cp -r  ../hello-world/hello-numpy-sag {provision_dir}/{username}/transfer/.\n",
+    "! cp -r  ../hello-world/hello-numpy-sag {provision_dir}/{username}/transfer/."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a084592-e023-406a-92cb-ba5d0b96897f",
+   "metadata": {},
+   "source": [
+    "We will use ```tree``` command view the ```transfer``` directory. \n",
+    "\n",
+    "```tree``` can be installed on linux system via ```sudo apt install tree```\n",
+    "\n",
+    "you can use ```ls -al``` if ```tree``` is not available \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b162ce0e-e219-475d-a014-9a7a23145239",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
     "! tree ../hello-world/hello-numpy-sag {provision_dir}/{username}/transfer/"
```

Comment:
is this tree command viewing `../hello-world/hello-numpy-sag`? 
or `{provision_dir}/{username}/transfer/`?

when you pass two params into tree, what does it do?